scripts/openwrt_build.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-export OPENWRT_DIR=/home/justin/repos/althea-firmware/build
+export OPENWRT_DIR=CHANGEME
 export TOOLCHAIN=toolchain-mipsel_24kc_gcc-7.3.0_musl
 export TARGET_NAME=target-mipsel_24kc_musl
```

Comment:
I wonder if we could ask it to pick the OpenWRT build's current toolchain somehow

Reply:
you could try to parse it out of the .config....